megalut/sim/stampgrid.py

```diff
@@ -214,9 +217,9 @@ def drawimg(catalog, simgalimgfilepath="test.fits", simtrugalimgfilepath=None, s
 		# We draw the desired profile
 		profile_type = params.profile_types[row["tru_type"]]
 		if profile_type == "Sersic":
-			gal = galsim.Sersic(n=float(row["tru_sersicn"]), half_light_radius=float(row["tru_rad"]), flux=float(row["tru_flux"]))
+			gal = galsim.Sersic(n=float(row["tru_sersicn"]), half_light_radius=float(row["tru_rad"]), flux=float(row["tru_flux"]), gsparams=gsparams)
```

Comment:
We have to specify `gsparams` every time? gsparams isn't some kind of global config?